workspaces/liferay-sample-workspace/client-extensions/liferay-sample-editor-config-contributor/client-extension.yaml

```diff
@@ -0,0 +1,9 @@
+assemble:
+    - from: build
+      into: static
+liferay-sample-editor-config-contributor:
+    editorConfigKeys:
+        - sampleClassicEditor
```

Comment:
With this, we target [Classic Editor sample](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-ckeditor-sample-web/src/main/java/com/liferay/frontend/editor/ckeditor/sample/web/internal/editor/configuration/ClassicEditorConfigContributor.java#L25).

Reply:
good choice, we don't want this sample to have any side effect on "production" editors but just sample ones